angular-pagedown.js

```diff
@@ -21,6 +21,7 @@ angular.module("ui.pagedown", [])
         restrict: "E",
         scope: {
             content: "=",
+            placeholder: "=",
```

Comment:
I don't need bi-directional, I wasn't imagining the placeholder changing at all.  Thanks for the spot - I'll change this.